packages/xo-server-test/src/_xoConnection.js

```diff
@@ -111,6 +113,23 @@ class XoConnection extends Xo {
   }
 
   async createTempBackupNgJob(params) {
+    // mutate and inject default values
+    defaultsDeep(params, {
+      mode: 'full',
+      name: getDefaultName(),
+      settings: {
+        '': {
+          // it must be enabled because the XAPI might be not able to coalesce VDIs
+          // as fast as the tests run
+          //
+          // see https://xen-orchestra.com/docs/backup_troubleshooting.html#vdi-chain-protection
+          bypassVdiChainsCheck: true,
+
+          // it must be 'never' to avoid race conditions with the plugin `backup-reports`
+          reportWhen: 'never',
+        },
+      },
+    })
     const job = await this.call('backupNg.createJob', params)
```

Comment:
Like other API methods, `backupNg.createJob` should return the id of the created object, if that's the case, then that's a bug.

Reply:
It's a bug.
https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-server/src/api/backup-ng.js#L11
https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-server/src/xo-mixins/backups-ng/index.js#L739

Reply:
Let's fix this in the API then, because this will impact your tests.

Reply:
Should i do it in this PR? Note: changing it in the API will not break `xo-web` i just validate it.

Reply:
As you wish.